scripts/release-assets

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+cd $(dirname $0)
+
+cd ..
+
+mkdir -p released
+
+if ! [[ -d assets ]] || ! [[ -d charts ]]; then
+    echo "No assets to move to released/"
+    exit 0
+fi
+
+cp -R assets/ released/assets
+rm released/assets/README.md
```

Comment:
why no readme ?

Reply:
The README.md is meant for the assets/ directory, the released/assets/ directory should probably have its own README.md declared in the released/ directory instead. I'll add that README.md in to explain the purpose of this folder to the charts-build-scripts PR.